sqldb/rust/src/sqldb.rs

```diff
@@ -0,0 +1,206 @@
+// This file is generated automatically using wasmcloud-weld and smithy model definitions
+//
+
+#![allow(clippy::ptr_arg)]
+#[allow(unused_imports)]
+use async_trait::async_trait;
+#[allow(unused_imports)]
+use serde::{Deserialize, Serialize};
+#[allow(unused_imports)]
+use std::{borrow::Cow, string::ToString};
+#[allow(unused_imports)]
+use wasmbus_rpc::{
+    deserialize, serialize, Context, Message, MessageDispatch, RpcError, RpcResult, SendOpts,
+    Timestamp, Transport,
+};
+
+pub const SMITHY_VERSION: &str = "1.0";
+
+/// Columns in result set
+#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+pub struct Column {
+    /// Column name in the result
+    #[serde(default)]
+    pub name: String,
+    /// column ordinal
+    pub ordinal: u32,
+    /// Data type of the column
+    #[serde(default)]
+    pub ty: String,
```

Comment:
again, another super tiny nitpick, but historically I've seen folks either go with `typ` or `datatype` when they're unable to use the word `type` as a field name.

Reply:
changed to dbType (see comment in slack)